### ISSUE TITLE
route_vpn.yml

### ISSUE THREAD

**Issue by meandus** [gabstopper/ansible-stonesoft#5]
Hi David,

For RBVPN, it could be very interesting to choose tunnel interfaces and gw for both side (as you do in the playbook) But It could be also better to allow us to disable some tunnel like Internet ==> MPLS and choose which network is standby or active

do you have also a get_facts for rbvpn ?

Rémy.

**Comment by gabstopper** [comment]
I agree - I will add an enable option on a per tunnel basis. There is also a route_vpn_fact.yml example in the playbooks directory. Right now most elements will dump their raw json data as provided by SMC API. Currently the engine json is formatted to be like the playbook and i'm considering this for other element types as well but I see some value in having the raw json as well (not to mention it simplifies coding and will change as the SMC API changes without mods on my end).

**Comment by gabstopper** [comment]
Hi Remy,
For disabling the RBVPN as in the SMC, I've had to request a feature enhancement as this isn't appearing in the latest SMC API. I will update you once I have more info. Also, rbvpn facts are available in the playbooks directory. I have also just updated the route_vpn module as well. Please see new documentation. You will also need the latest smc-python dev version as well.
